space/core/components/issues/filters/priority.tsx

```
@@ -2,7 +2,9 @@
 
 import React, { useState } from "react";
 import { observer } from "mobx-react";
+import { useTranslation } from "@plane/i18n";
 // ui
+import { useTranslation } from "@plane/i18n";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useTranslation'. Consider to delete it or rename it.

'useTranslation' is defined here:


(lint/suspicious/noRedeclare)